include/Page/Base.php

```diff
@@ -230,6 +230,7 @@ public function modify_admin_footer_text( $footer_text ) {
 	 */
 	public function enqueue_script() {
 		wp_enqueue_script( 'bulk-move', $this->get_plugin_dir_url() . 'assets/js/src/bulk-move.js', array( 'jquery', 'postbox' ), BM_BulkMove::VERSION, true );
+		wp_enqueue_script( 'select2', $this->get_plugin_dir_url() . 'assets/js/select2.min.js', array( 'jquery' ), '4.0.0', true );
```

Comment:
Add this enqueue before `bulk-move` and then add `select2` as dependencies to `bulk-move`.